webserver/views/api_tools.py

```diff
@@ -122,18 +122,12 @@ def validate_listen(listen, listen_type):
                     log_raise_400("JSON document may not contain track_metadata.additional_info.tags "
                                    "longer than %d characters." % MAX_TAG_SIZE, listen)
         # MBIDs
-        if 'release_mbid' in listen['track_metadata']['additional_info']:
-            lmbid = listen['track_metadata']['additional_info']['release_mbid']
-            if not is_valid_uuid(lmbid):
-                log_raise_400("Release MBID format invalid.", listen)
-        if 'recording_mbid' in listen['track_metadata']['additional_info']:
-            cmbid = listen['track_metadata']['additional_info']['recording_mbid']
-            if not is_valid_uuid(cmbid):
-                log_raise_400("Recording MBID format invalid.", listen)
-        ambids = listen['track_metadata']['additional_info'].get('artist_mbids', [])
-        for ambid in ambids:
-            if not is_valid_uuid(ambid):
-                log_raise_400("Artist MBID format invalid.", listen)
+        single_mbid_keys = ['release_mbid', 'recording_mbid', 'release_group_mbid', 'track_mbid']
```

Comment:
You could easily have these two lines be one and skip the assignment. Don't fix this time, just going forward.